// dom-tools/src/index.js
/* Provides DOM Helpers for -IE8+- IE9+
 *
 * FIXME: Revisit this library and ensure all methods and/or variables are
 * compatible with IE8.
 *
 * Inspired by jQuery, this library tries to mirror most of jQuery API or at
 * least provide similar names.
 */

module.exports = {

  canUseDOM: require('./canUseDOM'),

  // jQuery inspired methods
  // -----------------------

  addClass: require('./addClass'),
  after: require('./after'),
  append: require('./append'),
  before: require('./before'),
  children: require('./children'),
  clone: require('./clone'),
  contains: require('./contains'),
  css: require('./css'),
  empty: require('./empty'),
  hasClass: require('./hasClass'),
  height: require('./height'),
  html: require('./html'),
  find: require('./find'),
  insertAfter: require('./insertAfter'),
  hide: require('./hide'),
  offset: require('./offset'),
  outerHeight: require('./outerHeight'),
  outerWidth: require('./outerWidth'),
  parent: require('./parent'),
  prepend: require('./prepend'),
  remove: require('./remove'),
  removeClass: require('./removeClass'),
  show: require('./show'),
  width: require('./width'),

  // jQuery methods
  // --------------

  nodeName: require('./nodeName'),

  // Events
  // ------

  addEventListener: require('./addEventListener'),
  removeEventListener: require('./removeEventListener'),

  __VERSION__: '0.1.4'
};
